src/containers/Widgets/Widgets.js

```diff
@@ -28,7 +28,10 @@ export default class Widgets extends Component {
   static reduxAsyncConnect(params, store) {
     const {dispatch, getState} = store;
     if (!isLoaded(getState())) {
-      return dispatch(loadWidgets());
+      const promise = dispatch(loadWidgets());
+      if (__SERVER__) {
```

Comment:
this definitely solves the problem, but it's a patch. For example, we have them everywhere in our app and it'd be too routine and not pretty to have `if (__SERVER__) {` everywhere. This _does_ show how to solve it though